client/package.json

```diff
@@ -1,25 +1,91 @@
 {
-  "name": "client",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "react": "^16.6.3",
-    "react-dom": "^16.6.3",
-    "react-scripts": "2.1.1"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": [
-    ">0.2%",
-    "not dead",
-    "not ie <= 11",
-    "not op_mini all"
-  ]
+	"name": "client",
+	"version": "0.1.0",
+	"private": true,
+	"dependencies": {
+		"axios": "^0.18.0",
+		"keymirror": "^0.1.1",
+		"polished": "^2.3.1",
+		"react": "^16.6.3",
+		"react-dom": "^16.6.3",
+		"react-redux": "^6.0.0",
+		"react-router-dom": "^4.3.1",
+		"react-scripts": "2.1.1",
+		"redux": "^4.0.1",
+		"styled-components": "^4.1.3"
+	},
+	"scripts": {
+		"start": "react-scripts start",
+		"build": "react-scripts build",
+		"test": "jest",
+		"eject": "react-scripts eject"
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "npm test && lint-staged && git add"
+		}
+	},
+	"lint-staged": {
+		"*.{js,jsx}": [
+			"prettier --config .prettierrc --write"
```

Comment:
Another element should be added to the array here, `git add` to add back any files that were staged.

Reply:
Fixed